### ISSUE TITLE
Remove BindTimeoutSeconds from KubeSchedulerConfiguration

### ISSUE THREAD

**Issue by cofyc** [kubernetes/kubernetes#90958]
`KubeSchedulerConfiguration.BindTimeoutSeconds` configures volume bind timeout for `VolumeBinding` plugin. It is not necessary after #83726.

- [x] Add versioned counterparts of `VolumeBindingArgs` and tests in `pkg/scheduler/apis/config/scheme/` https://github.com/kubernetes/kubernetes/issues/91139
- [ ] Eliminate `KubeSchedulerConfiguration.BindTimeoutSeconds` and configure via PluginConfig only

**Comment by cofyc** [comment]
cc @ahg-g @alculquicondor @Huang-Wei

**Comment by Huang-Wei** [comment]
SG. BTW: we won't introduce a v1alpha3 version - v1beta1 will be the next version.

/ref #89701.

**Comment by alculquicondor** [comment]
> Add versioned counterparts of VolumeBindingArgs and tests in pkg/scheduler/apis/config/scheme/

That should be part of the existing PR

**Comment by ahg-g** [comment]
Thanks @cofyc I think the order should be to first add the versioned counterparts in a separate PR, then remove the flag which I guess will happen as part of v1beta1

**Comment by ahg-g** [comment]
> > Add versioned counterparts of VolumeBindingArgs and tests in pkg/scheduler/apis/config/scheme/
> 
> That should be part of the existing PR

we can do it in a separate smaller PR for API reviewers.

**Comment by cofyc** [comment]
> Thanks @cofyc I think the order should be to first add the versioned counterparts in a separate PR, then remove the flag which I guess will happen as part of v1beta1

should I add the versioned counterparts in exiting v1alpha2 or new version v1alpha3?

**Comment by ahg-g** [comment]
The existing v1alpha2.

**Comment by alculquicondor** [comment]
#91420 is merged. You can remove `KubeSchedulerConfiguration.BindTimeoutSeconds` now
